src/components/EventFilters.svelte

```diff
@@ -2,16 +2,19 @@
 import {
   filteredMeetingsByAll,
   filteredMeetingsByLocation,
+  filteredMeetingsByMine,
   searchedMeetings,
 } from '../data/analytics'
 import { clearEventFilter } from '../data/eventFiltering'
-import { setFilters } from '../data/filtering'
+import { removeFilter, setFilters } from '../data/filtering'
 import LocationFilter from './LocationFilter.svelte'
 import SearchFilter from './SearchFilter.svelte'
+import ToggleFilter from './ToggleFilter.svelte'
 
 export let filter = {}
 
 $: location = filter.location.value || ''
+$: onlyMyEvents = filter.participant.active
```

Comment:
to avoid having to remind ourselves what `participant` means in the future, what would you thik of `participating`?